apps/site/assets/ts/shuttles/components/StationDropdown.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement, useState, useEffect } from "react";
+import renderSvg from "../../helpers/render-svg";
+import arrowIcon from "../../../static/images/icon-down-arrow.svg";
+import { ShuttleStop } from "./__shuttles";
+
+interface Props {
+  stations: ShuttleStop[];
+  onSelect: (stationId: string) => void;
+}
+
+const StationDropdown = ({
+  stations,
+  onSelect
+}: Props): ReactElement<HTMLElement> => {
+  const [selectedStationId, setSelectedStationId] = useState("");
+  useEffect(
+    () => {
+      onSelect(selectedStationId);
+    },
+    [onSelect, selectedStationId]
```

Comment:
@thecristen I think you may want to leave `onSelect` out of this list of triggers

Reply:
@phildarnowsky sadly the linter gives me a warning in that case